qiskit/transpiler/preset_passmanagers/level3.py

```diff
@@ -82,6 +81,8 @@ def level_3_pass_manager(pass_manager_config: PassManagerConfig) -> StagedPassMa
     routing_method = pass_manager_config.routing_method or "sabre"
     translation_method = pass_manager_config.translation_method or "translator"
     scheduling_method = pass_manager_config.scheduling_method
+    init_method = pass_manager_config.init_method
+    optimization_method = pass_manager_config.optimization_method
```

Comment:
the usual order:
```suggestion
    init_method = pass_manager_config.init_method
    routing_method = pass_manager_config.routing_method or "sabre"
    translation_method = pass_manager_config.translation_method or "translator"
    optimization_method = pass_manager_config.optimization_method
    scheduling_method = pass_manager_config.scheduling_method
```

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8305/commits/6d344c068c7e3605d735f3efa654b9f1e406a0cb